geode-core/src/main/java/org/apache/geode/internal/cache/wan/serial/SerialGatewaySenderEventProcessor.java

```diff
@@ -865,4 +868,29 @@ protected void enqueueEvent(GatewayQueueEvent event) {
     // @TODO This API hasn't been implemented yet
     throw new UnsupportedOperationException();
   }
+
+  public void sendBatchDestroyOperationForDroppedEvent(EntryEventImpl event) {
+    if (logger.isDebugEnabled()) {
+      logger.debug(
+          "SerialGatewaySenderEventProcessor sends BatchDestroyOperation to secondary for event {}",
+          event);
+    }
+
+    EntryEventImpl droppedEvent = EntryEventImpl.create((LocalRegion) this.queue.getRegion(),
+        Operation.DESTROY, 0, null/* newValue */, null, false, sender.getCache().getMyId());
+    droppedEvent.disallowOffHeapValues();
+    droppedEvent.setTailKey(-1L);
```

Comment:
Can you use the input event instead of creating a new one here? Also, I don't see where the input event's eventID is being used. Does it need to be?